### PR TITLE
Add conic projections to the projection gallery

### PR DESCRIPTION
**Pull request by willschlitzer**
Adding conic projections to the projection gallery, as requested in #292.

### CODE REVIEW

**Review by seisman — APPROVED** [review]
It looks good to me. 

Please click the "Ready for review" button if you have finished this PR (so I can merge).

File: examples/projections/conic/conic_equidistant.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+"""
+import pygmt
+
+fig = pygmt.Figure()
+fig.coast(
+    shorelines="1/0.5p",
+    region=[-88, -70, 18, 24],
+    projection="D-79/21/19/23/12c",
+    land="lightgreen",
+    water="lightblue",
```
Comment (weiji14) [inline examples/projections/conic/conic_equidistant.py:16]: Maybe we should use gray to keep it consistent across all the projection tutorials?

Comment (seisman) [reply examples/projections/conic/conic_equidistant.py:16]: Please see some discussions in #292.

Comment (weiji14) [reply examples/projections/conic/conic_equidistant.py:16]: Ah sorry, missed that thread. Ok to merge then.

File: examples/projections/conic/conic_lambert.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+``Llon0/lat0/lat1/lat2/width``: Give projection center ``lon0/lat0``, two standard
+parallels ``lat1/lat2``, and the map width.
+"""
+import pygmt
+
+fig = pygmt.Figure()
+fig.coast(
+    shorelines="1/0.5p",
+    region=[-130, -70, 24, 52],
+    projection="D-100/35/33/45/12c",
```
Comment (seisman) [inline examples/projections/conic/conic_lambert.py:14]: Typo?
```suggestion
    projection="L-100/35/33/45/12c",
```

**Review by seisman — APPROVED** [review]

**Pull request merged**